datamover/pkg/drivers/https/migration.go

```diff
@@ -49,19 +42,19 @@ var JOB_RUN_TIME_MAX = 86400           //seconds, equals 1 day
 var s3client osdss3.S3Service
 var bkendclient backend.BackendService
 
-var logger = log.New(os.Stdout, "", log.LstdFlags)
-
 const WT_DOWLOAD = 48
 const WT_UPLOAD = 48
```

Comment:
@sfzeng Since we don't have sub-process like  download and upload here in migration.go, so this 2 constant `WT_DOWNLOAD` and `WT_UPLOAD` should be removed.

Reply:
Done, thanks !